tests/DotNotationHelperTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\BLT\Tests\Helpers;
+
+use Blumilk\BLT\Helpers\DotNotationHelper;
+use PHPUnit\Framework\TestCase;
+
+class DotNotationHelperTest extends TestCase
+{
+    public function testGetValueUsingDotNotation(): void
+    {
+        $array = [
+            "user" => [
+                "profile" => [
+                    "email" => "user@example.com",
+                    "age" => 25,
+                ],
+            ],
+        ];
+
+        $this->assertEquals("user@example.com", data_get($array, "user.profile.email"));
+        $this->assertEquals(25, data_get($array, "user.profile.age"));
+        $this->assertNull(data_get($array, "user.profile.name"));
+    }
+
+    public function testReturnsNullWithEmptyPath(): void
```

Comment:
If you change tests' name in one file, please be consistent and change in other files too.